src/styles.css

```diff
@@ -0,0 +1,15 @@
+#canvas {
+  background-color: silver;
+  border: solid 5px black;
+}
+
+#draw-button {
+  width: 150px;
```

Comment:
we should strive to use responsive units (em, rem, %, etc.) as much as possible unless we know, across all device types, that a size should be fixed.